code/circuit.py

```diff
@@ -0,0 +1,24 @@
+class circuit:
+
+    # a list of lines. The line should be in order
+    connection = []
+
+    #Constructors
+    def __init__(self):
+        self.connection = []
```

Comment:
Keep this line. This line is all you need to declare and initialize a field in Python.